### ISSUE TITLE
Live Errors

### ISSUE THREAD

**Issue by Spuds** [elkarte/Elkarte#3592]
In the editor code (since 1.0) we have an option for ```live_errors``` in the $editorOptions array which then will enable the ```errorbox_handler``` class in ```elk_script.js```  Thing is that I can't see anywhere that was ever enabled in 1.0, 1.1 or 2.0?  Not sure if it was at some point and got lost in some refactoring or ???  @emanuele45 I think that is your code (because it has no comments 🤣 ) It seems to work but I'm not really sure of all the things it was supposed to do? So really the question is what to do with it and what happened to it.

**Comment by emanuele45** [comment]
Funnily ehough, this was something Norv added https://github.com/elkarte/Elkarte/commit/ab6fd51 and I removed https://github.com/elkarte/Elkarte/commit/b20c76e ?
TBH I'm not even sure what it was about right now. lol

**Comment by emanuele45** [comment]
Ooohh!
That one!
That one was cool indeed, TBH I thought it was already working like this, but probably we replaced it with ajax calls and return of an error instead of?

**Comment by emanuele45** [comment]
I mean, the idea there was to avoid having to reload the page when an error occurred and load the error message somewhere directly in the editor.
I have to check again the code because I completely forgot everything about the editor... 🤦‍♂️

**Comment by Spuds** [comment]
```I completely forgot everything about the editor``` I 🙏 for that day 😄 

I'll leave it in place for you to take a look.  When I did the quick test it did not activate until after post was pressed ... I'm not sure if it sent and returned or blocked the send and showed the errors.  The attachment errors are not hooked into it that I am aware, would only want too many or over total limits there.

If we keep it, it should be redone properly as a plugin, like mentions, drafts, emoji, undo, etc

**Comment by Spuds** [comment]
I got it somewhat working, but find it a disruptive, with the error showing and hiding it causes the page to jump around.  

Firing off an ajax check when hitting post seems like a more robust idea, you are not nagging the user until they are actually doing something wrong.  Easier to extend to other checks as well.

**Comment by Spuds** [comment]
I reworked this a bit so ... we will see.

- I updated the addEvent helper to allow for easy off when adding a new event.  Right now its a hardcoded namespace but this is only used in one place AFAIK
- It will not check for an error until the first attempt at post, and if in error block.  The old way was to unwieldy,  for example if you stared to type something into the subject, and then backspaced due to an initial typo, blamo, error box.  Message box was just as bad and left for a very poor user experience.
- Changed it to use infobox not errorbox so its not yelling at anyone
- Cut down errorbox_handler to bare essentials.  The original could deal with multiple errors for the same input, this one can not, its just a single error that will be displayed.  It also added its own events and had several `evals` to do the checking.  Now the checking must be done in the caller function, and errors added/removed by the caller.  So basically errorbox_handler just deals with add/removing errors to some errorbox on the page and then showing/hiding that box as needed.
